OJP/OJP_ModesSupport.xsd

```diff
@@ -246,7 +246,7 @@
 			</xs:enumeration>
 			<xs:enumeration value="pooling">
 				<xs:annotation>
-					<xs:documentation>In some cases pooling is not an ALTERNATIVE MODE OF OPERATION, but is better served with a TimedLeg. Then this MODE OF OPERATION is to be used.</xs:documentation>
+					<xs:documentation>In some cases pooling is not an ALTERNATIVE MODE OF OPERATION, but is better served with a TimedLeg, Then this MODE OF OPERATION is to be used.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>In some cases pooling is not an ALTERNATIVE MODE OF OPERATION, but is better served with a TimedLeg. Then this MODE OF OPERATION is to be used.</xs:documentation>
```

Reply:
I think the original version is more grammatically correct.

Reply:
There were issues with replacing e.g. which touched all "Leg."